app/assets/stylesheets/frontend/print/_html-publication.scss

```diff
@@ -53,4 +53,20 @@
       padding: 0;
     }
   }
+
+  #meta-data {
```

Comment:
Like previous comment about the contact field, this might want to be called `print-meta-data` so it's more obvious what it is (`meta-data` is a pretty generic name and could be re-used on other elements).